vita3k/renderer/src/scene.cpp

```diff
@@ -168,4 +168,121 @@ COMMAND(handle_draw) {
     }
     }
 }
+
+COMMAND(handle_transfer_copy) {
+    const uint32_t colorKeyValue = helper.pop<uint32_t>();
+    const uint32_t colorKeyMask = helper.pop<uint32_t>();
+    const SceGxmTransferColorKeyMode colorKeyMode = helper.pop<SceGxmTransferColorKeyMode>();
+    const SceGxmTransferImage *images = helper.pop<SceGxmTransferImage *>();
+    const SceGxmTransferImage *src = &images[0];
+    const SceGxmTransferImage *dest = &images[1];
+    const SceGxmTransferType src_type = helper.pop<SceGxmTransferType>();
+    const SceGxmTransferType dst_type = helper.pop<SceGxmTransferType>();
+
+    const auto src_type_is_linear = src_type == SCE_GXM_TRANSFER_LINEAR;
+    const auto src_type_is_tiled = src_type == SCE_GXM_TRANSFER_TILED;
+    const auto src_type_is_swizzled = src_type == SCE_GXM_TRANSFER_SWIZZLED;
+    const auto dest_type_is_linear = dst_type == SCE_GXM_TRANSFER_LINEAR;
+    const auto dest_type_is_tiled = dst_type == SCE_GXM_TRANSFER_TILED;
+    const auto dest_type_is_swizzled = dst_type == SCE_GXM_TRANSFER_SWIZZLED;
+
+    if (src_type_is_linear && dest_type_is_linear) {
```

Comment:
```suggestion
    if (src_type && dst_type ) {
```
remove all const auto before, some gamee using two type same type is compatible